### DISCUSSION TITLE
Not receiving verification email in my student email

### DISCUSSION THREAD

**Discussion by P-Sood** [community/community#34228] [Other Feature Feedback, Questions, & Ideas]
Hi i have added both aliases to my student email and have been wanting Github Pro for over a year now, but it just never sends. Looking for some help! 

There is another similar discussion but it feels weird to backpack off of it at #23527

**Comment by DSDmark** [comment]
Hey,
About for your  pro title, I think you should apply it again with your email normal  which you use normally with any official document of your school or collage.
And also it not takes that much time like one of half year for title so, just try and try for it.
Maximum it takes 1 week for verifying your document or email.

# There are several reasons why your pro title is Rejected

- The item you uploaded is insufficient to demonstrate your current academic status. Modifications to your photo proof might have been detected by the system. Please see suggestions.
Suggested proof to expedite your approval:

- School issued photo ID with current enrollment date.

- Schedule for current enrollment with school name.

- Signed registration certificate with current enrollment date.

- Transcripts with current enrollment date.
